src/api/init.ts

```diff
@@ -242,6 +242,9 @@ export async function startApiServer(opts: {
       setResponseNonCacheable(res);
     }
     if (error && !res.headersSent) {
+      if (error instanceof RespError) {
+        return res.status(error.status).send({ error: error.message });
+      }
       res.status(500);
       const errorTag = uuid();
       Object.assign(error, { errorTag: errorTag });
```

Comment:
The `expressWinston.errorLogger` also needs configured so that it doesn't emit `level: error` logs for these. Something like...
```ts
  app.use(
    expressWinston.errorLogger({
      winstonInstance: logger as winston.Logger,
      metaField: (null as unknown) as string,
      blacklistedMetaFields: ['trace', 'os', 'process'],
      skip: (_req, _res, error) => {
        // Do not log errors for client 4xx responses
        return error instanceof RespError;
      },
    })
  );
```

Reply:
Well, so far it's not emitting any error like that because I return after sending the response. Logs are after the error handler middleware. But I'll write the skip script instead. That makes more sense.